TESTING.md

```diff
@@ -476,20 +477,48 @@ To open the context menu:
 2. Create new chat
 3. User doesn't see model dropdown
 
-### Enterprise account
+### Cody Enterprise - Cody Gateway Instance
 
 #### Chat
 
-1. Login to Cody Enterprise account
-2. Create new chat
-3. User should see the default model in the dropdown but is unable to change it
+1. Login to the Cody Enterprise Gateway Instance
+2. Create a new chat
+3. User should not be able to change the LLM model on the chat window
+4. Ask Cody a question in the new chat window. 
 
 #### Commands
 
-1. Login to Cody Enterprise account
+1. Login to the Cody Enterprise Cody Gateway Instance
 2. Go to commands panel
 3. Trigger command
-4. Command should be executed with default model
+4. Command should be executed
+
+#### Completions
+
+1. Login to the Cody Enterprise Cody Gateway Instance
+2. Generate a completion using the manual-trigger key binding on any primary languages (Javascript, Typescript, TypescriptReact, Python, Go).
+ 
+
+### Cody Enterprise - AWS Bedrock Instance 
+
+#### Chat
+
+1. Login to the Cody Enterprise AWS Bedrock Instance
+2. Create a new chat
+3. User should not be able to change the LLM model on the chat window
+4. Ask Cody a question in the new chat window. 
+
+#### Commands
+
+1. Login to the Cody Enterprise AWS Bedrock Instance
+2. Go to commands panel
+3. Trigger command
+4. Command should be executed
+
+#### Completions
+
+1. Login to the Cody Enterprise AWS Bedrock Instance
+2. Generate a completion using the manual-trigger key binding on any primary languages (Javascript, Typescript, TypescriptReact, Python, Go).
```

Comment:
This would be better if you removed the duplication and said something like: There are these kinds of instances (list them) and then had one set of instructions, like "For each kind of instance: ..."

Reply:
One thing to note is that we are evaluating the LLM dropdown for Enterprise customers using Cody Gateway, which might be not work with the consolidation. I agree combining them reads better, but I'm not sure if the team is going through a check list and having it split up by instances is easier in that case. If we do consolidate, shouldn't this also apply to the entire document? Do the following with these instances: Free, Pro, Enterprise 1, and 2? Will require a rewrite of the whole thing and I don't know why we didn't do that in the first place.